src/pygeoprocessing/routing/routing.pyx

```diff
@@ -793,10 +804,10 @@ def fill_pits(
                 'pixels complete')
 
         # search block for locally undrained pixels
-        for yi in range(1, win_ysize+1):
-            yi_root = yi-1+yoff
-            for xi in range(1, win_xsize+1):
-                xi_root = xi-1+xoff
+        for yi in range(win_ysize):
+            yi_root = yi+yoff
+            for xi in range(win_xsize):
+                xi_root = xi+xoff

```

Comment:
I'm assuming this is where you "fixed a prior bug". Could you just comment why it was iterating from `1` before and why we don't want to do that now? I'd love to know the context for this fix / change.

Reply:
I just noticed this loop ran from 1 to n+1 but then inside the parameter was always i-1. so just changed it from 0 to n then just use i. There's no functional change here.